invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/BackofficeRecordEditor/BackofficeRecordEditor.js

```diff
@@ -0,0 +1,37 @@
+// EditUserDialog.js
```

Comment:
IMHO, I would get rid of this completely and just have a component inside Backoffice -> Series -> `NewSeries` or `EditSeries`, and they internally render a `SeriesForm` that is filled with values when editing.